configs/server.toml

```diff
@@ -1,122 +1,337 @@
 enabled = true
+
+# Network address and port for the QUIC server.
+# For example, "0.0.0.0:8080" binds to all interfaces on port 8080.
 address = "0.0.0.0:8080"
+
+# Maximum number of simultaneous bidirectional streams in QUIC.
 max_concurrent_bidi_streams = 10_000
+
+# Size of the buffer for sending datagrams in QUIC.
 datagram_send_buffer_size = 100_000
+
+# Initial Maximum Transmission Unit (MTU) for QUIC connections.
 initial_mtu = 8_000
+
+# Size of the sending window in QUIC, controlling data flow.
 send_window = 100_000
+
+# Size of the receiving window in QUIC, controlling data flow.
 receive_window = 100_000
+
+# Interval in milliseconds for sending keep-alive messages in QUIC.
 keep_alive_interval = 5_000
+
+# Maximum idle time in milliseconds before a QUIC connection is closed.
 max_idle_timeout = 10_000
 
+# QUIC certificate configuration.
 [quic.certificate]
+# Indicates whether the QUIC certificate is self-signed.
+# `true` for self-signed certificates, often used in internal or testing environments.
+# `false` for certificates issued by a certificate authority, common in production.
 self_signed = true
+
+# Path to the QUIC TLS certificate file.
 cert_file = "certs/iggy_cert.pem"
+
+# Path to the QUIC TLS key file.
 key_file = "certs/iggy_key.pem"
 
+# Message cleaner configuration.
 [message_cleaner]
+# Enables or disables the background process for deleting expired messages.
+# `true` activates the message cleaner.
+# `false` turns it off, messages will not be auto-deleted based on expiry.
 enabled = true
+
+# Interval in seconds for running the message cleaner.
 interval = 60
 
+# Message saver configuration.
 [message_saver]
+# Enables or disables the background process for saving buffered data to disk.
+# `true` ensures data is periodically written to disk.
+# `false` turns off automatic saving, relying on other triggers for data persistence.
 enabled = true
+
+# Controls whether data saving is synchronous (enforce fsync) or asynchronous.
+# `true` for synchronous saving, ensuring data integrity at the cost of performance.
+# `false` for asynchronous saving, improving performance but with delayed data writing.
 enforce_fsync = true
+
+# Interval in seconds for running the message saver.
 interval = 30
 
+# Personal access token configuration.
 [personal_access_token]
+# Sets the maximum number of active tokens allowed per user.
 max_tokens_per_user = 100
 
+# Personal access token cleaner configuration.
 [personal_access_token.cleaner]
+# Enables or disables the token cleaner process.
+# `true` activates periodic token cleaning.
+# `false` disables it, tokens remain active until manually revoked or expired.
 enabled = true
+
+# Interval in seconds for running the token cleaner.
 interval = 60
 
+# System configuration.
 [system]
+# Base path for system data storage.
 path = "local_data"
 
+# Database configuration.
 [system.database]
+# Path for storing database files.
```

Comment:
It might be worth mentioning that it's relative to `system.path`, other than that everything else looks good :)

Reply:
Updated